src/App.php

```diff
@@ -38,14 +38,14 @@ class App
 
     /** @var array|false Location where to load JS/CSS files */
     public $cdn = [
-        'atk' => 'https://ui.agiletoolkit.org/public', // develop branch
+        'atk' => 'https://cdn.jsdelivr.net/gh/atk4/ui@2.2/public',
```

Comment:
wrong for `develop`

there should be no PRs like this, instead:

1. autocommit `App::$version` update below from release script to `develop` directly

2. update CDN (this commented line) in release process locally and release with built js/css files